package.json

```diff
@@ -33,6 +33,7 @@
     "cypress:run:debug": "node ./scripts/debug.js cypress:run",
     "cypress:verify": "cypress verify --dev",
     "dev": "gulp dev",
+    "dev:vite": "CYPRESS_INTERNAL_VITE_DEV=1 yarn dev",
```

Comment:
What does `CYPRESS_INTERNAL_VITE_DEV` do?

Reply:
It launches the app/launchpad with vite enabled. It's slower on initial startup and has a few kinks when new dependencies are detected but you got HMR which is nice when doing anything UI related. The default is a rollup build with a watcher

Reply:
nice! I was wondering how to enable HMR when running these.